### DISCUSSION TITLE
将body解析为CopyResp失败: error 1: expected value at line 1 column 1

### DISCUSSION THREAD

**Discussion by budongbujiling** [lanyeeee/copymanga-downloader#19] [Q&A] [answered]
大佬，新版本0.3.2搜索提示 0: 获取用户信息失败，将body解析为CopyResp失败: error 1: expected value at line 1 column 1，完全不懂怎么用，求求求

**Comment by lanyeeee** [comment]
搜索不会报这个错
这个错误是 `获取用户信息失败`，每次启动时都会自动获取，失败了影响的功能是
- 右上角的头像和用户名显示
- 收藏夹

其他功能不影响
如果搜索功能也用不了，**大概率是网络问题，换个代理试试**

另外你给的报错信息是完整的吗？截个图看看？

**Reply by budongbujiling** [reply]
大佬，不会用代理。。。我是在2024年12月2日开始使用的0.3.1版本。不用代理，不用登录用户，直接在搜索漫画就可以找到相关漫画，下载速度非常快。后续就是在12号左右，下载速度变慢，直到无法搜素。github不会用，不知道能否上传截图。截图分别是在漫画搜素出现 0: 搜索漫画失败，将body解析为CopyResp失败: error
1: expected value at line 1 column 1、用户登录 0: 使用账号密码登录失败，将body解析为CopyResp失败: error
1: expected value at line 1 column 1、TOKEN 0: 获取用户信息失败，将body解析为CopyResp失败: error
1: expected value at line 1 column 1 输入数据后出现的提示。感谢大佬回复。

**Reply by budongbujiling** [reply]
![20241222105724](https://github.com/user-attachments/assets/41f661f3-2249-42c3-8543-35a8f0d75a0d)

**Reply by lanyeeee** [reply, marked as answer]
这是网络问题，解决方案是挂代理

**Reply by budongbujiling** [reply]
感谢大佬，尝试用了一下代理，果然没问题了，多谢多谢。就是之前不知道咋回事，不用代理也没问题，速度还飞快。
![20241222194451](https://github.com/user-attachments/assets/6a13b9c1-15e5-4d6c-afb2-5cf880c448f0)
